pkgs/tools/security/sirikali/default.nix

```diff
@@ -0,0 +1,88 @@
+{ lib
+, stdenv
+, qtbase
+, libpwquality
+, hicolor-icon-theme
+, fetchFromGitHub
+, wrapQtAppsHook
+, cmake
+, pkg-config
+, libgcrypt
+, cryfs
+, encfs
+, fscrypt-experimental
+, gocryptfs
+, securefs
+, sshfs
+, libsecret
+, kwallet
+, noKWallet ? false
+, noLibsecret ? false
```

Comment:
Could you change these to avoid double negation? I think we should have `withKWallet ? true` as a default rather than the other way round!